doc/src/manual/distributed-computing.md

```diff
@@ -1267,7 +1267,7 @@ requirements for the inbuilt `LocalManager` and `SSHManager`:
     running the Julia REPL (i.e., the master) with the rest of the cluster on the cloud, say on Amazon
     EC2. In this case only port 22 needs to be opened at the remote cluster coupled with SSH client
     authenticated via public key infrastructure (PKI). Authentication credentials can be supplied
-    via `sshflags`, for example ```sshflags=`-i <keyfile>` ```.
+    via `sshflags`, for example * sshflags=`-i <keyfile>` *.
```

Comment:
The original Markdown is correct, though, and the updated version seems like a step back.

Reply:
yes, we need to fix on the DocumenterVitepress side, I couldn't figure out the right escaping sequence in order to get something that vitepress can render. @asinghvi17, here at the bottom is the expected code by vitepress: https://luxdl.github.io/DocumenterVitepress.jl/v0.0.18/markdown-examples#Escaping-characters

Reply:
Just as a note, and related to https://github.com/JuliaLang/julia/pull/53988#discussion_r1555122577 as well, but having to sort out the escaping for everything etc. is a major drawback of the Markdown -> AST -> Markdown -> HTML approach DocumenterVitepress is taking. Getting away from all that was the major reason why we transitioned away from MkDocs in the first place back in the day.